skymp5-server/ts/systems/login.ts

```diff
@@ -66,9 +67,14 @@ export class Login implements System {
           this.log("Logged as " + res.data.user.id);
         }
       });
+    } else if (this.offlineMode === true && gameData && gameData.profileId) {
```

Comment:
I think we need to check `typeof gameData.profileId`
the user can pass non-number values. please also test it